icu4c/source/common/utrie_swap.cpp

```diff
@@ -281,7 +281,6 @@ ucptrie_swap(const UDataSwapper *ds,
             return 0;
         }
     }
-
```

Comment:
Please revert this file, now that the only net change is removing this one empty line.